capstone-project/src/main/angular-webapp/src/app/info-window/info-window.component.ts

```diff
@@ -31,7 +31,8 @@ export class InfoWindowComponent implements OnInit {
     this.animal = animalValue;
     this.description = descriptionValue;
     this.reporter = reporterValue;
-    this.submitEvent.emit({animal: animalValue, description: descriptionValue, reporter: reporterValue})
+    const idToken = (this.user ? this.user.idToken : "")
```

Comment:
The parentheses are redundant. 

You can use optional chaining operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining